include/Graphs/GenericGraph.h

```diff
@@ -512,10 +506,13 @@ struct GraphTraits<Inverse<SVF::GenericNode<NodeTy,EdgeTy>* > >
     typedef NodeTy NodeType;
     typedef EdgeTy EdgeType;
 
-    typedef edge_unary_function<EdgeType*, NodeType*> DerefEdge;
+    static inline NodeType* edge_dest(const EdgeType* E)
```

Comment:
I see. Could we use the name edge_src here?

Reply:
Yes, but I would advise against changing it. I chose `edge_dest()` because it's the suggested name in the [GraphTraits template](https://github.com/llvm/llvm-project/blob/8e29b4b323b87f3855dc71abf1e3f3d48952a4e4/llvm/include/llvm/ADT/GraphTraits.h#L66). Hence, graph algorithms might use `edge_dest()` to dereference an edge returned by `child_edge_begin()` (which is not yet implemented for `GenericNode`, but still).

I think in the context of an inverse traversal of `GenericNode`, the *destination* of an edge is the edge's source node, hence `edge_dest()` still fits.